File: ServiceTimeRecord/ServiceTimeRecord/Models/Task.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;

namespace ServiceTimeRecord.Models
{
    [Table("Tasks")]
    public class Task
    {
        [Key]
        public int id { get; set; }
        [Required]
        [MaxLength(40)]
        public string name { get; set; }
        [Required]
        public string description { get; set; }
        public ICollection<Time> times { get; set; }
        public Employee employee { get; set; }
        public int employeeId { get; set; }
    }
}

File: ViewTimeRecord/src/app/model/task.ts
export class Task {
    constructor(id = null, name = '', description = '', employeeId = null){
        this.id = id,
        this.name = name,
        this.description = description,
        this.employeeId = employeeId
    }
    id: number;
    name: string;
    description: string;
    employeeId: number;
}
